src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js

```diff
@@ -56,8 +56,13 @@ export class DocumentEItems extends Component {
           <DocumentEItemUrls eitems={eitems} />
         ) : (
           <>
-            No e-resources currently available. <br />
-            <Link to={this.onClickEItemRequestLink()}>Request the e-book.</Link>
+            <p>No e-resources currently available.</p>
+            <Link
+              to={this.onClickEItemRequestLink()}
+              className="default-margin-top"
+            >
+              <Button fluid>Request the e-book</Button>
```

Comment:
I think there is a shorter way ```Link as={Button}``` (to be checked)

Reply:
From the documentation it does not seem that way https://v5.reactrouter.com/web/api/Link

Reply:
then it was Button as={Link}, we have a lot of usage of these in the code